k8s/upgrades/jiva-0.6.0-0.8.1/jiva_volume_upgrade.sh

```diff
@@ -113,10 +113,17 @@ if [[ -z $c_name ]]; then
 fi
 
 controller_version=`kubectl get deployment $c_dep -n $ns -o jsonpath='{.metadata.labels.openebs\.io/version}'`
-
+if [[ "$controller_version" != "" ]] && [[ "$controller_version" != "$target_upgrade_version" ]]; then
```

Comment:
our idea is to allow if controller_version is empty or controller_version != 0.8.1.
So, if controller_version is 0.6.0, upgrade should happen.
But, with the condition here, upgrade won't happen if controller_version is 0.6.0
So, change the condition to:
` if [[ "$controller_version" != "" ]] && [[ "$controller_version" == "$target_upgrade_version" ]]; then`

Reply:
fixed